src/main/java/cn/nukkit/exception/ChatColorCharOutOfBoundsException.java

```diff
@@ -0,0 +1,7 @@
+package cn.nukkit.exception;
+
+public class ChatColorCharOutOfBoundsException extends Exception {
+    public ChatColorCharOutOfBoundsException(String message) {
+      super(message);
+    }
+}
```

Comment:
Is it's unused now that the Bukkit's chat formatting enum was removed? If it is, please remove it.